packages/mdc-button/_mixins.scss

```diff
@@ -127,6 +128,8 @@
 }
 
 @mixin mdc-button__icon_ {
+  @include mdc-rtl-reflexive-box(margin, right, 8px);
```

Comment:
We should probably remove the `margin-right: 8px` line below (line 136)

Reply:
sorry, my bad.